src/Math-Numerical/Integer.extension.st

```diff
@@ -1,5 +1,54 @@
 Extension { #name : #Integer }
 
+{ #category : #'*Math-Numerical' }
+Integer >> findNK [
+	" Reverse binomial coefficient. Answer a <Collection> with all n and k such that n take: k = self. Elements in the answered Collection should be read as paired. Each pair represents (n,k) in the binomial coefficient formula. 
+	See https://math.stackexchange.com/a/103385/205 for details. "
+
+	| k |
+	[ self isInteger and: [ self > 1 ] ] assert.
```

Comment:
Given you've defined this in the Integer class, do you need to have the `isInteger` check?

Reply:
You're right! I removed the check. Thanks!